src/main/kotlin/net/atos/client/smartdocuments/exception/SmartDocumentsBadRequestResponseExceptionMapper.kt

```diff
@@ -13,6 +13,6 @@ class SmartDocumentsBadRequestResponseExceptionMapper : ResponseExceptionMapper<
         status == Response.Status.BAD_REQUEST.statusCode
 
     override fun toThrowable(response: Response): SmartDocumentsBadRequestException = SmartDocumentsBadRequestException(
-        "SmartDocuments responded with a 400 Bad Request: ${response.location} (${response.statusInfo})"
+        "User not known to SmartDocuments"
```

Comment:
I think this is a bit too "short" a corner you take here.

I'd prefer a specific exception for this situation - as long as we can identify that - or otherwise, add it as a possibility to the end of the message that's there already.

Reply:
Added the suggestion to check user in SD at the end of 400 Bad Request message

Reply:
See conversation with @infonl-marcel below for use-cases of the code.